packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,44 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
-  meetingAdapter.onStateChange((newState) => {
-    setCurrentPage(newState.page);
-    setCurrentMeetingState(newState.meeting?.state);
-  });
-
+  const [unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);
   const [showChat, setShowChat] = useState(false);
   const [showPeople, setShowPeople] = useState(false);
 
+  useEffect(() => {
```

Comment:
Moving the `onStateChange` subscriber inside a `useEffect` seems unrelated to the intent of this PR. Please extract into its own (far far simpler) PR.